site/en/getstarted/prerequisite-helm.md

```diff
@@ -20,30 +20,41 @@ Before you install Milvus, check your hardware and software to see if they meet
 | CPU                 | Intel CPU Sandy Bridge or later                              |<ul><li>standalone: 8 core or more</li><li>cluster: 16 core or more</li></ul>| Current version of Milvus does not support AMD and Apple M1 CPUs. |
 | CPU instruction set | <ul><li>SSE4.2</li><li>AVX</li><li>AVX2</li><li>AVX-512</li></ul> |<ul><li>SSE4.2</li><li>AVX</li><li>AVX2</li><li>AVX-512</li></ul> |  Vector similarity search and index building within Milvus require CPU's support of single instruction, multiple data (SIMD) extension sets. Ensure that the CPU supports at least one of the SIMD extensions listed. See [CPUs with AVX](https://en.wikipedia.org/wiki/Advanced_Vector_Extensions#CPUs_with_AVX) for more information.                           |
 | RAM                 | <ul><li>standalone: 16G</li><li>cluster: 64G</li></ul>       |<ul><li>standalone: 32G</li><li>cluster: 128G</li></ul>        | The size of RAM depends on the data volume.                  |
-| Hard drive          | SATA 3.0 SSD or higher                                       |SATA 3.0 SSD or higher | The size of hard drive depends on the data volume.           |
-
+| Hard drive          | SATA 3.0 SSD or higher                                       |NVMe SSD or higher | The size of hard drive depends on the data volume.           |
 
 ## Software requirements
 
-It is recommended that you run the Kubernetes cluster on Linux platforms. 
+It is recommended that you run the Kubernetes cluster on Linux platforms.
 
 kubectl is the command-line tool for Kubernetes. Use a kubectl version that is within one minor version difference of your cluster. Using the latest version of kubectl helps avoid unforeseen issues.
 
 minikube is required when running Kubernetes cluster locally. minikube requires Docker as a dependency. Ensure that you install Docker before installing Milvus using Helm. See <a href="https://docs.docker.com/get-docker">Get Docker</a> for more information.
 
-
-
 | Operating system | Software                                                     | Note                                                         |
 | ---------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
 | Linux platforms  | <ul><li>Kubernetes 1.16 or later</li><li>kubectl</li><li>Helm 3.0.0 or later</li><li>minikube (for Milvus standalone)</li><li>Docker 19.03 or later (for Milvus standalone)</li></ul> | See [Helm Docs](https://helm.sh/docs/) for more information. |
 
-| Software | Version                       |
-| -------- | ----------------------------- |
-| etcd     | 3.5.0                         |
+| Software | Version                       | Note |
+| -------- | ----------------------------- | ---- |
+| etcd     | 3.5.0                         |  See [additional disk requirements](###-additional-disk-requirements). |
```

Comment:
You need to add a pipe at the end of the other two rows below to form a complete table.
See https://www.markdownguide.org/extended-syntax/#tables

Reply:
Error rectified.